src/scenes/Projects/Members/Update/UpdateProjectMember.tsx

```diff
@@ -55,16 +55,29 @@ export const UpdateProjectMember = ({
   });
 
   const [deleteProjectMember] = useMutation(DeleteProjectMemberDocument, {
-    update: callAll(
-      removeItemFromList({
-        listId: 'projectMembers',
-        item: { id: projectMemberId },
-      }),
-      removeItemFromList({
-        listId: [project, 'team'],
-        item: { id: projectMemberId },
-      })
-    ),
+    update: (cache) => {
+      callAll(
+        removeItemFromList({
+          listId: 'projectMembers',
+          item: { id: projectMemberId },
+        }),
+        removeItemFromList({
+          listId: [project, 'team'],
+          item: { id: projectMemberId },
+        })
+      );
+
+      if (data?.session.user?.id === userId) {
+        invalidateProps(cache, project, [
+          'partnerships',
+          'team',
+          'rootDirectory',
+          'narrativeReports',
+          'financialReports',
+          'budget',
+        ]);
+      }
+    },
```

Comment:
`removeItemFromList` is expecting to be given directly to the `update` option (or merged via `callAll`). Right now this function isn't even being called.
You need to do
```ts
removeItemFromList({...})(cache, ...);
```

Also I think the second removeItemFromList isn't needed if we are removing the whole object.